src/components/Calculator.js

```diff
@@ -1,76 +1,262 @@
-import { Component } from "react";
-
+import {Component} from "react";
 import Number from "./Number";
 import Operator from "./Operator";
 import Screen from "./Screen";
+import {Container, Row, Col} from 'react-bootstrap';
 
-class Calculator extends Component {
-  state = {
-    first: null,
-    operator: null,
-    second: null,
-  };
-
-  handleNumberClick = (number) => {
-    if (!this.state.operator) {
-      this.setState({ first: `${this.state.first || ""}${number}` });
-    } else {
-      this.setState({ second: `${this.state.second || ""}${number}` });
+
+const Styles = {
+    wrapper: {
+        maxWidth: '380px',
```

Comment:
```suggestion
        maxWidth: 380,
```
You can drop the `px` when using inline styling. React adds them as default.